client/test/SimiliarListings.test.js

```diff
@@ -0,0 +1,26 @@
+import SimilarListings from '../src/index.jsx';
+import React from 'react';
+import sinon from 'sinon';
+import { shallow, mount, render } from 'enzyme';
+import "isomorphic-fetch";
+
+
+describe('SimilarListings component renders similar listings correctly', () => {
+  it('renders correctly', () => {
+    const SimilarListingWrapper = shallow(
+      <SimilarListings currentListingId={1} />
+    );
+    expect(SimilarListingWrapper).toMatchSnapshot();
+  });
```

Comment:
I was thinking about that too! But since it's a module that I"m using to make the carousel, I assume they have their own tests to make things work >.>